Start-at-new-page/README.md

```diff
@@ -0,0 +1,29 @@
+# Replace merge field with HTML string using Mail merge in C#
+
+This example illustrates how to generate records in each page during mail merge.
+
+# How to run the project
+
+1. Download this project to a location in your disk.
+
+2. Open the solution file using Visual Studio.
+
+3. Rebuild the solution to install the required NuGet packages.
+
+4. Run the application.
+
+# Screenshots
+
+By running this application, you will get the output Word document as follows.
+
+<p align="center">
+<img src="Images/Start-at-new-page-output.png" alt="Start-at-new-page-output"/>
+</p>
+
+To replace merge fields with HTML string, design your template Word document with the required content and merge fields using Microsoft Word as follows.
```

Comment:
To start each record in new page, design the template document with the corresponding group start and group end present in the text body of the Word document as follows.

Reply:
Addressed.